src/drive/mobile/lib/background.js

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: 0 */
```

Comment:
I don't really like global ignores since it means we may forgot `console.*` in this file but if it's a temporary solution until we remove all these console?

Reply:
This is !